app/client/src/common/components/index.js

```diff
@@ -4,7 +4,7 @@
 
 import Divider from './Divider'
 import Icon from './Icon'
-import LabeledInput, { Label, Input } from './LabeledInput'
+import LabeledInput, { Label, Input } from '../../features/form/components/fragments/LabeledInput'
```

Comment:
This line can probably be removed entirely from here now. This was mainly a convenience file to make it easier to `import` multiple components from `common/components` at once. But since it isn't a part of `common/components` anymore, it probably shouldn't be there.

However, this would mean that you would need to make sure to not `import` the `Label` or `Input` components from `common/components` anymore – they should both just be directly imported from `LabeledInput.js`.